package.json

```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "@jest/globals": "29.7.0",
+    "@types/jest": "^29.5.11",
```

Comment:
not a pinned dep, also why is this needed here? I don't see any other TS changes anywhere.

Reply:
I can delete this but i dont know where its coming from... To your point no TS changes.